tests/libvmifact/factory.go

```diff
@@ -154,3 +156,9 @@ func NewWindows(opts ...libvmi.Option) *kvirtv1.VirtualMachineInstance {
 	vmi.Spec.Domain.Firmware = &kvirtv1.Firmware{UUID: WindowsFirmware}
 	return vmi
 }
+
+func WithEchoHelloNoUserData() cloudinit.NoCloudOption {
```

Comment:
I can only repeat that I don't like the function name. The important thing here is that it's non-empty.
I'd be happier to see `func WithNonEmptyUserData() libvmi.Option`, as that's what we actually need. But I'm sure others can accept the current proposal.